app/src/main/java/com/realityexpander/tasky/core/presentation/notifications/RemindAtNotificationManagerImpl.kt

```diff
@@ -77,15 +79,13 @@ object RemindAtNotificationManagerImpl : IRemindAtNotificationManager {
     }
 
     override fun showNotification(
-        context: Context,
         alarmIntent: Intent
     ) {
         alarmIntent.apply {
             val agendaItem = getParcelableExtra(ALARM_NOTIFICATION_INTENT_EXTRA_AGENDA_ITEM) as? AgendaItem
 
             agendaItem?.let { item ->
                 createNotification(
-                    context,
                     alarmId = alarmIntent.getIntExtra(ALARM_NOTIFICATION_INTENT_EXTRA_ALARM_ID, 0),
                     agendaItemTypeStr = item.toAgendaItemTypeStr(),
```

Comment:
Here you could pass the item without converting it to string (I'll explain the why in the next comment)